api/app/models/text.rb

```diff
@@ -199,6 +205,21 @@ def to_s
 
   private
 
+  def title_changed?
+    reload_title
```

Comment:
Are there any suggestions on how to do this that doesn't require reloading every time _or_ that can be a little smarter about when to reload?  The `title` method get the value from the main title association, which will have changed during `title=`.  

Right now, to compare the old value with the current I need to refresh that association.